research-software-directory/RSD-as-a-service
# 560 order organisations

# Allow custom order of participating organisations for software and projects

Closes #560

Changes proposed in this pull request:
*  Previously participating organisations were ordered alphabetically on the software and project page. The users expressed need to define display order freely. To achieve that we implemented similar drag-and-drop approach as with contributors and team members    

How to test:
* `make start` this will remove old containers and database, build the solution, generate random data and start the solution
* open rsd on localhost and login as rsd_admin so you can edit existing software and project
* navigate to software and start editing it. You can add new organsation(s) and change their position
* confirm that position is applied on software page
* do same exercise on edit project page and confirm that organisation order is applied

## Example order organisations on edit software page
![image](https://user-images.githubusercontent.com/9204081/195691950-55d35e23-e31c-4e85-b545-ea0947edafca.png)

## Example order organisations on edit project page
![image](https://user-images.githubusercontent.com/9204081/195692482-2c61e55c-720e-4adf-893c-19ce536c3fbd.png)

PR Checklist:
